patches/index.html.patch

```diff
@@ -0,0 +1,28 @@
+--- a/assets/index.html.bak	2022-09-06 10:53:02.326980516 -0400
++++ b/assets/index.html	2022-09-06 11:03:20.726980280 -0400
+@@ -1,12 +1,19 @@
+ <!DOCTYPE html>
+ <html lang="en">
+-	<head>
+-		<meta charset="UTF-8" />
+-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+-		<title>Discord Test Client</title>
++	 	<head>
++ 		<meta charset="UTF-8" />
++ 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
++		<title>Fosscord Staging</title>
++ 		<link rel="stylesheet" href="/assets/fosscord.css" />
++ 		<link id="logincss" rel="stylesheet" href="/assets/fosscord-login.css" />
++ 		<link id="customcss" rel="stylesheet" href="/assets/user.css" />
+ 		<link rel="stylesheet" href="/assets/fosscord.css" />
+-		<link id="logincss" rel="stylesheet" href="/assets/fosscord-login.css" />
+-		<link id="customcss" rel="stylesheet" href="/assets/user.css" />
++		<!-- <meta property="og:title" content="Fosscord" /> -->
++		<meta name="description" content="Fosscord Staging Fosscord Instance.">
++		<meta property="og:description" content="Fosscord [Fosscord Test Client]" />
++		<meta property="og:url" content="https://staging.fosscord.com/app" />
++		<link rel="icon" type="image/png" href="/assets/Fosscord.png" />
++		<meta property="og:image" content="/assets/Fosscord.ico" />
+ 		<!-- inline plugin marker -->
+ 		<!-- preload plugin marker -->
+ 	</head>
```

Comment:
is there any reason to include this file? Youve already applied the patch to index.html.